go/cmd/dolt/commands/profile.go

```diff
@@ -51,10 +52,11 @@ Remove the profile named {{.LessThan}}name{{.GreaterThan}}.`,
 }
 
 const (
-	addProfileId        = "add"
-	removeProfileId     = "remove"
-	GlobalCfgProfileKey = "profile"
-	DefaultProfileName  = "default"
+	addProfileId          = "add"
+	removeProfileId       = "remove"
+	GlobalCfgProfileKey   = "profile"
+	DefaultProfileName    = "default"
+	defaultProfileWarning = "Default profile has been added. All dolt commands taking global arguments will use this default profile until it is removed.\nWARNING: This will alter the behavior of command which specify no `--profile`.\nIf you are using dolt in contexts where you expect a `.dolt` directory to be accessed, the default profile will be used instead."
```

Comment:
```suggestion
	defaultProfileWarning = "Default profile has been added. All dolt commands taking global arguments will use this default profile until it is removed.\nWARNING: This will alter the behavior of commands which specify no `--profile`.\nIf you are using dolt in contexts where you expect a `.dolt` directory to be accessed, the default profile will be used instead."
```